Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gem "hoe"
+gem "minitest"
+gem "rake"
+gem "ruby_parser", "3.13.1"
+gem "flog", "~> 4.6"
+
+group :test do
+  gem "rspec"
+  gem "simplecov"
+end
```

Comment:
could you add a new line character, please ?

Reply:
done 👍